File: DevUIArea/angular-product-setup/src/commonComponents/models/requests/products/attributesForProductOptionRequest.ts
import { AttributeModel } from '../../products/attributeModel';
import { ProductModel } from '../../products/productModel';
import { ProductOptionModel } from '../../products/productOptionModel';


export class AttributesForProductOptionRequest {
    public product: ProductModel;
    public productOption: ProductOptionModel;
    public attributes: AttributeModel[];

    constructor() {
        this.product = new ProductModel();
        this.productOption = new ProductOptionModel();
        this.attributes = new Array<AttributeModel>();
    }
}

File: Service/OrderInvoiceService/Models/API/Products/Requests/AttributesForProductOptionRequest.cs
// ***********************************************************************
// Assembly         : OrderInvoiceService
// Author           : Admin
// Created          : 02-22-2021
//
// Last Modified By : Admin
// Last Modified On : 02-22-2021
// ***********************************************************************
// <copyright file="AttributesForProductOptionRequest.cs" company="OrderInvoiceService">
//     Copyright (c) . All rights reserved.
// </copyright>
// <summary></summary>
// ***********************************************************************
namespace OrderInvoiceService.Models.API.Products.Requests
{
    /// <summary>
    /// Class AttributesForProductOptionRequest.
    /// </summary>
    public class AttributesForProductOptionRequest
    {
        /// <summary>
        /// Gets or sets the product.
        /// </summary>
        /// <value>The product.</value>
        public ProductModel Product { get; set; }
        /// <summary>
        /// Gets or sets the product option.
        /// </summary>
        /// <value>The product option.</value>
        public ProductOptionModel ProductOption { get; set; }
        /// <summary>
        /// Gets or sets the attributes.
        /// </summary>
        /// <value>The attributes.</value>
        public AttributeModel[] Attributes { get; set; }
    }
}
